### PR TITLE
Close #2625 by updating method to enable Dependabot security updates

### LINKED ISSUE #2625: Update methodology for enabling Dependabot security updates
### What article on docs.github.com is affected?
https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/configuring-dependabot-security-updates

### What part(s) of the article would you like to see updated?
[Managing Dependabot security updates for your repositories](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories)

### Additional information
I think the current methodology for enabling Dependabot security updates, part of which is in [`assets/images/help/repository/enable-dependabot-security-updates-drop-down.png`](https://github.com/github/docs/blob/main/assets/images/help/repository/enable-dependabot-security-updates-drop-down.png), is out of date. I think I might have a fix for this.

image

### PR DESCRIPTION
**Pull request by hirehamir**
### Why:
As mentioned in #2625, [the documentation for enabling automated Dependabot security updates](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories) appears to be out-of-date.

### What's being changed:
I'm changing steps two, three and four of the ["Managing Dependabot security updates for your repositories" section of the "Configuring Dependabot security updates" page](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories) to match what I think is the new user interface for enabling Dependabot security updates.

In this pull request, I removed what I think is an out-of-date screenshot and replaced it with a screen capture that I think matches the current user interface.

Original:

current appearance of "Managing Dependabot security updates for your repositories" section of the "Configuring Dependabot security updates" page

**Appearance with changes from this pull request:**

appearance of "Managing Dependabot security updates for your repositories" section of the "Configuring Dependabot security updates" page with changes from this pull request

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@hamirmahal Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Comment by hamirmahal** [comment]
@janiceilene You're welcome! Happy to help.

**Comment by hubwriter** [comment]
@hamirmahal - Many thanks for raising this PR. That's a good catch and a great fix. 👍 

I'm going to make a slight modification just to make it clear that the procedure refers to enabling/disabling Dependabot security updates on a single repository, and also to match the instructions we give in the related "[Managing security and analysis settings for your repository](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-security-and-analysis-settings-for-your-repository#enabling-or-disabling-security-and-analysis-features-for-private-repositories)" topic.

Thanks again for helping to keep GitHub documentation up to date. Your assistance is very much appreciated!

**Comment by hubwriter** [comment]
See my changes to this PR in this commit: https://github.com/github/docs/pull/2662/commits/5a0a1b153bc601368837350051cfb9d645f61a00

Staged deployment of the updated page is here: https://docs-2662--fixdependabot-updat.herokuapp.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/configuring-dependabot-security-updates#managing-dependabot-security-updates-for-your-repositories

**Review by hubwriter — APPROVED** [review]

**Comment by hamirmahal** [comment]
@hubwriter You're welcome! Happy to help.

I think there's a typo at the very end of [line 56 in `content/github/managing-security-vulnerabilities/configuring-dependabot-security-updates.md`](https://github.com/github/docs/commit/5a0a1b153bc601368837350051cfb9d645f61a00#diff-6fbc0bce5aec6ccafdecd960dd728bb8564e9aa957fe0d10abad120ca2dcdc0aR56).

I think fixing the typo is my only suggested change. [I gave a couple of alternatives](https://github.com/github/docs/commit/5a0a1b153bc601368837350051cfb9d645f61a00#r45978854).

Thanks for the direct preview link! That saves me a bit of time so I don't have to find the page again, since I didn't bookmark the preview link.

File: content/github/managing-security-vulnerabilities/configuring-dependabot-security-updates.md [hunk 1]
```diff
@@ -42,17 +42,19 @@ If security updates are not enabled for your repository and you don't know why,
...
+#### Enabling or disabling {% data variables.product.prodname_dependabot_security_updates %} for an individual repository
+
 {% data reusables.repositories.navigate-to-repo %}
-{% data reusables.repositories.sidebar-security %}
-{% data reusables.repositories.sidebar-dependabot-alerts %}
-1. Above the list of alerts, use the drop-down menu and select or unselect **{% data variables.product.prodname_dependabot %} security updates**.
-  ![Drop-down menu with the option to enable {% data variables.product.prodname_dependabot_security_updates %}](/assets/images/help/repository/enable-dependabot-security-updates-drop-down.png)
+{% data reusables.repositories.sidebar-settings %}
+{% data reusables.repositories.navigate-to-security-and-analysis %}
+1. Under "Configure security and analysis features", to the right of "{% data variables.product.prodname_dependabot %} security updates", click or **Enable** or **Disable**.
```
Comment (hubwriter) [inline content/github/managing-security-vulnerabilities/configuring-dependabot-security-updates.md:56]:
```suggestion
1. Under "Configure security and analysis features", to the right of "{% data variables.product.prodname_dependabot %} security updates", click **Enable** or **Disable**.
```

**Review by hubwriter — APPROVED** [review]
@hamirmahal - typo corrected, thanks.

I'll get this merged now.

**Comment by hamirmahal** [comment]
@hubwriter You're welcome! Thanks for merging.

**Pull request merged**